src/Avalonia.Base/Threading/DispatcherTimer.cs

```diff
@@ -200,7 +192,10 @@ public void Stop()
             }
         }
 
-        
+        protected virtual void Dispose(bool disposing)
```

Comment:
There's no need for the `Dispose(bool)` pattern when there's no finalizer. Just make `Dispose()` virtual if needed.